Lunch/CreateAccountViewController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  CreateAccountViewController.swift
+//  Lunch
+//
+//  Created by Spencer Amann on 8/1/15.
+//  Copyright (c) 2015 Spencer Amann. All rights reserved.
+//
+
+import UIKit
+import Parse
+
+class CreateAccountViewController: UIViewController {
+
+    @IBOutlet weak var userNameTextField: UITextField!
+    @IBOutlet weak var emailTextField: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+    @IBOutlet weak var confirmedPasswordTextField: UITextField!
+
+    let loginIdentifier = "loginAfterCreateAccount"
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    @IBAction func createAccountButtonClick(sender: AnyObject) {
+        let user = PFUser()
+
+        var noErrors = checkkInputForErrors(userNameTextField.text ?? "", password: passwordTextField.text ?? "",
+                                        email: emailTextField.text ?? "", confirmedPassword: confirmedPasswordTextField.text ?? "")
+        if noErrors {
+            user.username = userNameTextField.text
+            user.password = passwordTextField.text
+            user.email = emailTextField.text
+            
+            user.signUpInBackgroundWithBlock { (succes: Bool, error:NSError?) -> Void in
+                if error == nil {
+                    self.performSegueWithIdentifier(self.loginIdentifier, sender: self)
+                } else {
+                    var alert = UIAlertView(title: "Error", message: "There was a problem", delegate: nil, cancelButtonTitle: "Whoops")
+                    alert.show()
+                }
+            }
+        }
+    }
+
+    func checkkInputForErrors(username: String, password: String, email: String, confirmedPassword: String) -> Bool {
+        var alert = UIAlertView()
```

Comment:
It's really weird that you're creating an instance of `UIAlertView` that you just overwrite, you probably meant `var alert: UIAlertView?`